### PR TITLE
DOCS-7870-made edits

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 1]
```diff
@@ -100,6 +100,65 @@ For example, type `cd /home/_your-user-name_/.bashrc`) in a text editor.
 . Save the `.bashrc` file.
 . Restart your terminal.
 
+== Preparing to Install Instio
```
Comment (dosten) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]:
```suggestion
== Preparing to Install Istio
```

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 2]
```diff
@@ -100,6 +100,65 @@ For example, type `cd /home/_your-user-name_/.bashrc`) in a text editor.
...
+
+When you install Instio in your environment, you must configure Instio settings based on the Instio version that you install.
```
Comment (dosten) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]:
```suggestion
When you install Istio in your environment, you must configure Istio settings based on the Istio version that you install.
```

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 3]
```diff
@@ -100,6 +100,65 @@ For example, type `cd /home/_your-user-name_/.bashrc`) in a text editor.
...
+
+=== Installing Instio 1.4x
```
Comment (dosten) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]:
```suggestion
=== Installing Istio 1.4.x
```

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 4]
```diff
@@ -100,6 +100,65 @@ For example, type `cd /home/_your-user-name_/.bashrc`) in a text editor.
...
+
+To install and configure Instio 1.4x for Anypoint Service Mesh:
```
Comment (dosten) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]:
```suggestion
To install and configure Istio 1.4.x for Anypoint Service Mesh:
```

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 5]
```diff
@@ -100,6 +100,65 @@ For example, type `cd /home/_your-user-name_/.bashrc`) in a text editor.
...
+
+. Download and install Instio using the Instio documentation.
```
Comment (dosten) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]:
```suggestion
. Download and install Istio using the Istio documentation.
```

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 6]
```diff
@@ -100,6 +100,65 @@ For example, type `cd /home/_your-user-name_/.bashrc`) in a text editor.
...
+. Enable the policy control flag:
++
+`set values.global.disablePolicyChecks=false`
```
Comment (dosten) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]:
```suggestion
`--set values.global.disablePolicyChecks=false`
```

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 7]
```diff
@@ -100,6 +100,65 @@ For example, type `cd /home/_your-user-name_/.bashrc`) in a text editor.
...
++
+For more information about this flag, see https://archive.istio.io/v1.4/docs/tasks/policy-enforcement/enabling-policy/[Policy Enforcement]. 
+
+=== Installing Instio 1.5x
```
Comment (dosten) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]:
```suggestion
=== Installing Istio 1.5.x
```

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 8]
```diff
@@ -100,6 +100,65 @@ For example, type `cd /home/_your-user-name_/.bashrc`) in a text editor.
...
+
+To install and configure Instio 1.5x for Anypoint Service Mesh:
```
Comment (dosten) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]:
```suggestion
To install and configure Istio 1.5.x for Anypoint Service Mesh:
```

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 9]
```diff
@@ -100,6 +100,65 @@ For example, type `cd /home/_your-user-name_/.bashrc`) in a text editor.
...
+
+. Download and install Instio using the Instio documentation.
```
Comment (dosten) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]:
```suggestion
. Download and install Istio using the Istio documentation.
```

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 10]
```diff
@@ -100,6 +100,65 @@ For example, type `cd /home/_your-user-name_/.bashrc`) in a text editor.
...
+components:
+  citadel:
+    enabled: true
+  telemetry:
+    enabled: true
+----
+If you configured your profile correctly with these configurations settings, a full manifest file is displayed as illustrated in the following example:
++
+----
+[source,text,linenums]
```
Comment (dosten) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]:
```suggestion
[source,text,linenums]
----
```

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 11]
```diff
@@ -100,6 +100,65 @@ For example, type `cd /home/_your-user-name_/.bashrc`) in a text editor.
...
+      enabled: true
+    v2:
+      enabled: false
+components:
+  citadel:
+    enabled: true
+  telemetry:
+    enabled: true
+----
+If you configured your profile correctly with these configurations settings, a full manifest file is displayed as illustrated in the following example:
```
Comment (tbruno) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Hi, 
You changed the meaning of what I had placed. "An example of the full manifest file would be. If using it, remember to choose the correct profile."

See that there is a field that is called profile.

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 12]
```diff
@@ -100,6 +100,65 @@ For example, type `cd /home/_your-user-name_/.bashrc`) in a text editor.
...
++
+----
+[source,text,linenums]
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
```
Comment (tbruno) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc:157]:
```suggestion
spec:
  profile: default
  components:
    policy:
      enabled: true
    sidecarInjector:
      enabled: true
    citadel:
      enabled: true
    telemetry:
      enabled: true
  addonComponents:
    prometheus:
      enabled: false
  values:
    global:
      disablePolicyChecks: false
    telemetry:
      v1:
        enabled: true
      v2:
        enabled: false
```

It was missing the actual spec 🤔

File: modules/ROOT/pages/configure-service-mesh.adoc [hunk 13]
```diff
@@ -27,6 +27,22 @@ You can configure Anypoint Service Mesh using the command line interface (CLI) o
 * xref::configure-using-CLI.adoc[Using CLI]
 * xref::configure-using-CRD.adoc[Using CRD]
 
+== Configuring Response Headers for Anypoint Service Mesh
+
+Services protected by Anypoint Service Mesh return extra headers. These headers are added by Envoy and the gRPC adapter.
+
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-service-mesh.adoc]: Remove extra linespace?

File: modules/ROOT/pages/configure-service-mesh.adoc [hunk 14]
```diff
@@ -27,6 +27,22 @@ You can configure Anypoint Service Mesh using the command line interface (CLI) o
...
+
+You can remove these headers to avoid returning them to the originating client.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-service-mesh.adoc]: Should we say something about why or under what conditions the audience wants or needs to do this?

File: modules/ROOT/pages/configure-service-mesh.adoc [hunk 15]
```diff
@@ -27,6 +27,22 @@ You can configure Anypoint Service Mesh using the command line interface (CLI) o
...
+
+To remove the headers:
+
+. Log in to your Ingress server.
+. Search and remove the following headers added by Envoy:
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-service-mesh.adoc]:
```suggestion
. Search for and remove the following headers added by Envoy:
```

File: modules/ROOT/pages/configure-service-mesh.adoc [hunk 16]
```diff
@@ -27,6 +27,22 @@ You can configure Anypoint Service Mesh using the command line interface (CLI) o
...
+* x-envoy-upstream-service-time
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-service-mesh.adoc]:
```suggestion
* `x-envoy-upstream-service-time`
```

File: modules/ROOT/pages/configure-service-mesh.adoc [hunk 17]
```diff
@@ -27,6 +27,22 @@ You can configure Anypoint Service Mesh using the command line interface (CLI) o
...
+* x-envoy-decorator-operation
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-service-mesh.adoc]:
```suggestion
* `x-envoy-decorator-operation`
```

File: modules/ROOT/pages/configure-service-mesh.adoc [hunk 18]
```diff
@@ -27,6 +27,22 @@ You can configure Anypoint Service Mesh using the command line interface (CLI) o
...
+. Search and remove the following headers added by the gRPC adapter:
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-service-mesh.adoc]:
```suggestion
. Search for and remove the `x-mule-report` added by the gRPC adapter.
```

File: modules/ROOT/pages/configure-service-mesh.adoc [hunk 19]
```diff
@@ -27,6 +27,22 @@ You can configure Anypoint Service Mesh using the command line interface (CLI) o
...
+* x-mule-report
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-service-mesh.adoc]: Omit single-item bullets.
```suggestion
```

File: modules/ROOT/pages/download-and-install-service-mesh.adoc [hunk 20]
```diff
@@ -16,20 +16,20 @@ To download the Anypoint Service Mesh client, type the following command at the
 `curl -Ls http://anypoint.mulesoft.com/servicemesh/xapi/v1/install > asmctl && chmod +x asmctl`
 
 
-== Configure Your Environment Variable
+// == Configure Your Environment Variable
```
Comment (kbostrom) [inline modules/ROOT/pages/download-and-install-service-mesh.adoc:19]: Configure or update? 
Choose the more precise of the two and use it consistently throughout this section...which is commented out?

File: modules/ROOT/pages/download-and-install-service-mesh.adoc [hunk 21]
```diff
@@ -16,20 +16,20 @@ To download the Anypoint Service Mesh client, type the following command at the
...
-. Save the `.bashrc` file.
-. Restart your terminal.
+// . Open the `.bashrc` file in your home directory.
+// +
+// For example, type `cd /home/<your-user-name>/.bashrc` in a text editor.
+// . Add the following command to the last line of the file:
+// +
+// `export PATH="your-dir:$PATH"`, where *your-dir* is the directory you want to add.
+// . Save the `.bashrc` file.
+// . Restart your terminal.
```
Comment (kbostrom) [inline modules/ROOT/pages/download-and-install-service-mesh.adoc:32]: Does one actually restart the terminal, or just the program?

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 1 above)
Comment (kbostrom) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Be sure to fix this GLOBALLY. It looks like your reviewer has caught them all, but double-check after you update the PR.
Also, please check for and resolve reviewer comments BEFORE submitting PRs for editing. Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 2 above)
Comment (kbostrom) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: I don't understand the purpose of this sentence; it seems to be stating the obvious, and in a bit of a confusing way.
If you install a particular version, don't the settings come with that version?
Are you trying to say that the audience needs to prepare its environment for the particular version of Istio it is installing? Or that different versions require different prep work?

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 3 above)
Comment (kbostrom) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: GLOBALLY search for this version typo as well.

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 5 above)
Comment (kbostrom) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: This topic is titled PREPARING to install, not installing. I think you need to retitle the container topic....

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 1 above)
Comment (kbostrom) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: After reading further, I see that this topic is NOT about PREPARING to install; it is about installing different versions of Instio. Please clarify.

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 2 above)
Comment (kbostrom) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: I now see that this is not about preparing but about how different Istio versions are installed in slightly different ways. Clarify this for the audience.

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 22]
```diff
@@ -100,6 +100,65 @@ For example, type `cd /home/_your-user-name_/.bashrc`) in a text editor.
...
+=== Installing Instio 1.4x
+
+To install and configure Instio 1.4x for Anypoint Service Mesh:
+
+. Download and install Instio using the Instio documentation.
+. Enable the policy control flag:
++
+`set values.global.disablePolicyChecks=false`
++
+For more information about this flag, see https://archive.istio.io/v1.4/docs/tasks/policy-enforcement/enabling-policy/[Policy Enforcement].
```
Comment (kbostrom) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Would your audience really need to look this up to perform this task? 
I think this is probably unnecessary.

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 11 above)
Comment (kbostrom) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: This sentence needs to be changed per reviewer comment.

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 12 above)
Comment (kbostrom) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc:157]: Please update per reviewer comment. 
I really can't edit something that continues to change.

**Review by kbostrom — APPROVED** [review]
Edited, but reviewer comments require addressing and remain unedited.

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 4 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 5 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 6 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 7 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 8 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 9 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 10 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 11 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 12 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc:157]: @tbruno the actual file you sent me was missing his portion. :-)

File: modules/ROOT/pages/configure-service-mesh.adoc (hunk 13 above)
Comment (priyaayyar) [reply modules/ROOT/pages/configure-service-mesh.adoc]: Yes, Done! Thanks!

File: modules/ROOT/pages/configure-service-mesh.adoc (hunk 15 above)
Comment (priyaayyar) [reply modules/ROOT/pages/configure-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/configure-service-mesh.adoc (hunk 16 above)
Comment (priyaayyar) [reply modules/ROOT/pages/configure-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/configure-service-mesh.adoc (hunk 17 above)
Comment (priyaayyar) [reply modules/ROOT/pages/configure-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/configure-service-mesh.adoc (hunk 18 above)
Comment (priyaayyar) [reply modules/ROOT/pages/configure-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/configure-service-mesh.adoc (hunk 19 above)
Comment (priyaayyar) [reply modules/ROOT/pages/configure-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 12 above)
Comment (tbruno) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc:157]: mmm, it is here: https://docs.google.com/document/d/1pwcuRVylXUvfj2LqkbYrEBk2j2YQKUv7UwwCWqjzo-o/edit

File: modules/ROOT/pages/download-and-install-service-mesh.adoc (hunk 20 above)
Comment (priyaayyar) [reply modules/ROOT/pages/download-and-install-service-mesh.adoc:19]: Done! Thanks!

File: modules/ROOT/pages/download-and-install-service-mesh.adoc (hunk 21 above)
Comment (priyaayyar) [reply modules/ROOT/pages/download-and-install-service-mesh.adoc:32]: Restart the terminal to launch the program again. Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 22 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: I'd leave it in for people who need more clarity about what the flag does. Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 11 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/configure-service-mesh.adoc (hunk 14 above)
Comment (priyaayyar) [reply modules/ROOT/pages/configure-service-mesh.adoc]: It is configured if the users don't want extra headed to be returned to the originating API. I think that comes across clearly. Done! Thanks!

**Pull request merged**